README.md

```diff
@@ -200,9 +200,59 @@ doxygen
 
 # Satori
 
-This repository also contains the Satori client for use with the [Satori Liveops Server](https://heroiclabs.com/satori/).
+Satori is a liveops server for games that powers actionable analytics, A/B testing and remote configuration. Use the Satori .NET Client to coomunicate with Satori from within your .NET game.
```

Comment:
I think the whole Satori section of content should come before the **Contribute** section.

Reply:
@novabyte good point, the section will be "nakama api" then "satori api", then "nakama contribute" and "satori contribute".